test/src/example.cpp

```diff
@@ -616,6 +617,11 @@ void Example::test_set_tileset(TileMap *p_tilemap, const Ref<TileSet> &p_tileset
 	p_tilemap->set_tileset(p_tileset);
 }
 
+bool Example::test_tween_smoke_test() {
+	Ref<Tween> tween = create_tween();
+	return tween.is_valid() && tween->is_class("Tween");
```

Comment:
Could it make sense to also check [`get_reference_count()`](https://docs.godotengine.org/en/stable/classes/class_refcounted.html#class-refcounted-method-get-reference-count)?